translations/fi.po

```diff
@@ -1030,3 +1054,127 @@ msgstr "Avaa kaikki"
 msgctxt "Accordion close all"
 msgid "Close all"
 msgstr "Sulje kaikki"
+
+msgctxt "Job search languages"
+msgid "Finnish"
+msgstr "Suomi"
+
+msgctxt "Job search languages"
+msgid "Swedish"
+msgstr "Ruotsi"
+
+msgctxt "Job search languages"
+msgid "English"
+msgstr "Englanti"
+
+msgctxt "Job search"
+msgid "jobs"
+msgstr "tehtävää"
+
+msgctxt "Job search"
+msgid "Application period ends"
+msgstr "Haku päättyy"
+
+msgctxt "Job search"
+msgid "Employment contract"
+msgstr "Palvelussuhde"
+
+msgctxt "Job search"
+msgid "Search"
+msgstr "Etsi"
+
+msgctxt "Events search"
+msgid "Use the format D.M.YYYY"
+msgstr "Käytä muotoa P.K.VVVV"
+
+msgctxt "Events search"
+msgid "Invalid start date"
+msgstr "Virheellinen aloituspäivämäärä"
+
+msgctxt "Events search"
+msgid "Invalid end date"
+msgstr "Virheellinen lopetuspäivämäärä"
+
+msgctxt "Events search"
+msgid "Selected start date is out of range with end date, setting end date to next day after start date."
```

Comment:
Is this string used outside the console.warn call? If not, this should probably be removed.